ui/Modal/ManageRemotes/PeerFollowRequest.svelte

```diff
@@ -0,0 +1,37 @@
+<script lang="typescript">
+  import { createEventDispatcher } from "svelte";
+
+  import type { Urn } from "../../src/urn";
+  import type { User } from "../../src/project";
+
+  import { Flex } from "../../DesignSystem/Primitive";
+  import { FollowToggle } from "../../DesignSystem/Component";
+
+  export let peer: User;
+  export let projectName: string;
+  export let projectUrn: Urn;
+
+  const dispatch = createEventDispatcher();
+
+  const firstSix = peer.peerId.substring(0, 7);
```

Comment:
This logic exists elsewhere as well, right? Should we put it in a shared location?

Reply:
I simplifier it to a one-liner. I think the duplication is okay for now.